data/card_catalogue/upgradeable-1.json

```diff
@@ -402,49 +511,38 @@
     "upgrade": "",
     "tier": "gold",
     "details": {
-      "rules": "A powerful hurricane is summoned over the play area which lasts as long as you want.",
-      "flavour": "It is rare for an area to know such devestation.",
+      "rules": {
+        "active": "A powerful hurricane is summoned over the play area for as long as you want.",
+        "passive": ""
+      },
+      "flavour": "It is rare for an area to know such devastation.",
       "image": "assets/core/card/illustration/Hurricane.png"
     }
   },
   {
     "title": "Barrier",
-    "upgrade": "Holomonk",
+    "upgrade": "",
     "tier": "blue",
     "details": {
-      "rules": "Summon an impassible wall of light on 1 square until end of turn.",
+      "rules": {
+        "active": "Summon an impenetrable column of light in adjacent square metre for 5 minutes.",
+        "passive": ""
+      },
       "flavour": "\"Quit the mime act and c'mon! We're getting out of here!\" - Barry the Bandit",
       "image": "assets/core/card/illustration/Barrier.png"
     }
   },
-  {
-    "title": "Holomonk",
-    "upgrade": "HoloKicker",
-    "tier": "green",
-    "details": {
-      "rules": "Summon an immobile monk until end of turn who punches characters who come within range",
-      "flavour": "His name is Po.",
-      "image": "assets/core/card/illustration/Holomonk.png"
-    }
-  },
-  {
-    "title": "HoloKicker",
-    "upgrade": "Mirror Image",
-    "tier": "red",
-    "details": {
-      "rules": "Summon 3 immobile martial artists until end of turn who knock characters who come within range back 8 squares",
-      "flavour": "Their names are Memboba, Imbosaa, and Steepedi.",
-      "image": "assets/core/card/illustration/HoloKicker.png"
-    }
-  },
   {
     "title": "Mirror Image",
     "upgrade": "",
     "tier": "gold",
     "details": {
-      "rules": "Summon a copy of any non-player character",
-      "flavour": "\"I hated the man so much I made a point of killing him twice\" - R.J. Alabaster, Deranged Summoner",
-      "image": "assets/core/card/illustration/HoloKicker.png"
+      "rules": {
+        "active": "Summon a copy of any NPC.",
+        "passive": ""
+      },
+      "flavour": "\"I hated the man so much I made a point of killing him twice\" - R.J. Alabaster, Deranged Cardmaster",
+      "image": "assets/core/card/illustration/Mirror Image.png"
```

Comment:
mild flavour change to suit 9c4ebfce1b3f2056d1a1ecd87f63877aa435474d and subsequent